code/game/objects/items/weapons/storage/toolbox.dm

```diff
@@ -70,13 +70,123 @@
 	origin_tech = list(TECH_COMBAT = 1, TECH_ILLEGAL = 1)
 	force = 7.0
 
-	New()
-		..()
-		var/color = pick("red","yellow","green","blue","pink","orange","cyan","white")
-		new /obj/item/weapon/screwdriver(src)
-		new /obj/item/weapon/wrench(src)
-		new /obj/item/weapon/weldingtool(src)
-		new /obj/item/weapon/crowbar(src)
-		new /obj/item/stack/cable_coil(src,30,color)
-		new /obj/item/weapon/wirecutters(src)
-		new /obj/item/device/multitool(src)
+/obj/item/weapon/storage/toolbox/syndicate/New()
+	..()
+	var/color = pick("red","yellow","green","blue","pink","orange","cyan","white")
+	new /obj/item/weapon/screwdriver(src)
+	new /obj/item/weapon/wrench(src)
+	new /obj/item/weapon/weldingtool(src)
+	new /obj/item/weapon/crowbar(src)
+	new /obj/item/stack/cable_coil(src,30,color)

```

Comment:
I know you've just de-intended the line but the whole pick-color business could be replaced with
``` /obj/item/stack/cable_coil/random(src,30) ```